spec/helpers/reminder_helper_spec.rb

```diff
@@ -0,0 +1,57 @@
+describe ReminderHelper do
+  name1 = 'name1'
+  name2 = 'name2'
+  reminder_names = [
+    name1, name2
+  ]
+
+  describe '#reminders?' do
+    it 'returns false when data has no active_reminder method' do
+      expect(reminders?({})).to be_falsey
```

Comment:
Why not explicitly check for whether it's true or not?

Reply:
Bad habit honestly. Since we are infact no interested in the nil case here checking true is the better choice hehe